og_access/src/OgAccessBundleFormAlter.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Drupal\og_access;
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\og\Og;
+use Drupal\Core\StringTranslation\TranslationInterface;
+
+/**
+ * Helper for og_access_form_alter().
+ */
+class OgAccessBundleFormAlter {
+  use StringTranslationTrait;
+
+  /**
+   * The entity bundle.
+   *
+   * @var string
+   */
+  protected $bundle;
+
+  /**
+   * The entity type ID.
+   *
+   * @var string
+   */
+  protected $entityTypeId;
+
+  /**
+   * The form entity which has been used for populating form element defaults.
+   *
+   * @var \Drupal\Core\Entity\EntityInterface
+   */
+  protected $entity;
+
+  /**
+   * Construct a BundleFormAlter object.
+   *
+   * @param \Drupal\Core\Entity\EntityInterface $entity
+   *   The entity object.
+   * @param Drupal\Core\StringTranslation\TranslationInterface $string_translation
+   *   The string translation object.
+   */
+  public function __construct(EntityInterface $entity, TranslationInterface $string_translation) {
+    $this->entity = $entity;
+    $this->stringTranslation = $string_translation;
```

Comment:
```suggestion
```

Reply:
I didn't notice earlier that we're instantiating OgAccessBundleFormAlter from within a hook_form_alter.

As we are injecting the service from the global container we might as well remove the Trait, or keep it as we might move to events instead of hooks in the future...